### PR TITLE
Juanvqz/change json to yaml format

### LINKED ISSUE #85: Increase readability of label data files
## Problem
Both `labels.json` and `skills.json` are not very readable for the following reasons:

- both are JSON files, which are not very readable for humans.
- currently the `labels.json` file uses emoji as unicode glyphs, which look like two codepoints in a string. It's not possible to identify the emojis from the text.

## Description
GitHub allows using ["colon-style markup"](https://docs.github.com/en/rest/reference/issues#update-a-label) instead of native emoji. We should use that as the value of the emoji key. Additionally YAML would be a much better option for files that are manually read or written.

## Alternatives
The current implementation works. Not working on the issue is the alternative.

## Implementation

- [ ] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by JuanVqz**
## Fixes

Fixes #85  by @dhruvkb 

## Description
change JSON to YAML format

## Checklist
 
- [ ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by JuanVqz** [comment]
@dhruvkb I didn't find the corresponding `\uD83D\uDFE5` emoji code in this [cheat sheet](https://www.webfx.com/tools/emoji-cheat-sheet/).
What should I do?

**Comment by dhruvkb** [comment]
@JuanVqz please leave them as codepoints for now. We can find replacements for them later.

**Comment by JuanVqz** [comment]
@dhruvkb  Would you check it, please? I think it is finished.

File: normalize_repos/labels.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+groups:
+  group:
+    name: "priority"
```
Comment (dhruvkb) [inline normalize_repos/labels.yml:3]: Values that are plain strings do not need to be surrounded in quotes.

File: normalize_repos/labels.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      label:
+        name: "ready for work"
+        color: "LIGHT"
+        description: "Ready for work"
+        emoji: "\uD83C\uDFC1"
+      label:
+        name: "blocked"
+        color: "MEDIUM"
+        description: "Blocked & therefore, not ready for work"
+        emoji: "\uD83D\uDEA7"
```
Comment (dhruvkb) [inline normalize_repos/labels.yml:37]: Please replace emojis that are present in the cheat sheet with the colon-representation. Only leave those that are absent as codepoints.
```suggestion
        emoji: ":construction:"
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Apart from the few changes pointed below, this is great work! However you will also need to make the corresponding changes to the Python code in `get_labels.py` that reads and interprets these files as that has been written for the JSON format.

**Comment by JuanVqz** [comment]
I don't know why the PR repo says unknown repository that is the reason I open another PR

**Pull request closed without merging**